express-api/src/typeorm/Entities/abstractEntities/Property.ts

```diff
@@ -57,7 +57,7 @@ export abstract class Property extends BaseEntity {
   @Column({
     type: 'point',
     transformer: {
-      to: (obj: GeoPoint) => `${obj.x},${obj.y}`, // Going to the database. Format into point.
+      to: (obj: GeoPoint) => `${obj?.x},${obj.y}`, // Going to the database. Format into point.
```

Comment:
Why the ? and why only on the first one?

Reply:
Crtl+Z mistake woops.